Magento_GiftCardAccount/templates/cart/block.phtml

```diff
@@ -70,7 +70,7 @@
                 "giftCard": {
                     "giftCardCodeSelector": "#giftcard-code",
                     "checkStatus": ".action.check",
-                    "giftCardStatusUrl": "<?= $getGiftCardStatusUrl ?>",
+                    "giftCardStatusUrl": "<?= $block->escapeHtmlAttr($getGiftCardStatusUrl) ?>",
```

Comment:
inside HtmlAttr please add escapejs (and test on storefront)

Reply:
Where in application I can test it ?